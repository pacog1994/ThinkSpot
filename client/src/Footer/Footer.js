import React, { Component } from 'react'
import { withStyles } from '@material-ui/core/styles'
//Container Component for different links - Log in/out, MySpot and home
class Footer extends Component {        
    render() {
        return (
            <div className={this.props.classes.root}>
                <a className={this.props.classes.link} href="https://docs.google.com/document/d/1Hurhlo6DY5P8jRT6IgXGiQwE6zbqF8OPE3QgY1I7hMs/edit?usp=sharing" target="_blank" rel="noopener noreferrer"> ThinkSpot Dev Notes</a>
            </div>
        )
    }    
}

const styles = theme => ({
    root: {
        marginTop: theme.spacing.unit * 4,
        height: theme.spacing.unit * 30,
        backgroundColor: "#242729"
    },
    link: {
        color: "white",
        textDecoration: 'none'
    }
})

export default withStyles(styles)(Footer)
