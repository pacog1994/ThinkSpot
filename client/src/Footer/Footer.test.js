import React from 'react'
import Footer from './Footer'
import renderer from 'react-test-renderer'

it('renders Footer', () => {
    const tree = renderer
        .create(<Footer/>)
        .toJSON()
    expect(tree).toMatchSnapshot()
})