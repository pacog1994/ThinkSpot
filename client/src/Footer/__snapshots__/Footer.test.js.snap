// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer 1`] = `
<div
  className="Footer-root-1"
>
  <a
    className="Footer-link-2"
    href="https://docs.google.com/document/d/1Hurhlo6DY5P8jRT6IgXGiQwE6zbqF8OPE3QgY1I7hMs/edit?usp=sharing"
    target="_blank"
  >
    ThinkSpot Dev Notes
  </a>
</div>
`;
